CHANGELOG.asciidoc

```diff
@@ -48,6 +48,7 @@ image::https://raw.githubusercontent.com/apache/tinkerpop/master/docs/static/ima
 * Server now supports WebSocket compression extension ([RFC7692](https://tools.ietf.org/html/rfc7692)).
 * Fixed bug with Bytecode serialization when `Bytecode.toString()` is used in Javascript.
 * Fixed "toString" for P and TextP to produce valid script representation from bytecode glv steps containing a string predicate in Javascript.
+* Fix bug which could cause Java driver to hang when using `ResultSet.statusAttributes()`
```

Comment:
nit: "Fixed" rather than "Fix" - we try to stick to a past tense with changelog entries. might want to quickly clean up a few of those "now supports" and "now delegates" above too.

Reply:
fixed